### ISSUE TITLE
Instability in Some Campaigns - Can't Save

### ISSUE THREAD

**Issue by resumeman** [SolastaMods/SolastaUnfinishedBusiness#4652]
I opened a similar issue earlier and it got closed. But it's persisting with the latest UB versions, and I verified that with UB uninstalled everything is fine.

When I start a new game with *some* but not all campaigns, I get the "Solasta has detected an instability" warning. The game starts normally, but I can't save the game ("Save" is greyed out, and F5 doesn't work).

I tried a variety of ways to start a game, all with the campaign Shire Child - Southern Belle. All the player logs are attached.

- Starting with 6 characters made by me, and everything set up as I normally do
- Starting with 6 premade characters
- Reducing the party size to 4 and starting with both 4 custom and 4 premade characters
- Disabling UB but leaving it installed
- Uninstalling UB
- Reinstalling UB and starting with I think 6 custom characters.

Uninstalling UB worked - with the mod uninstalled, it worked fine. But just turning it off didn't change anything.

This is *not* a problem with *every* campaign. In general, but not entirely, the issue seems to be with newer campaigns, with older ones working fine. But some new ones also work fine. Some campaigns where I encountered this issue are:

- Shire Child - Southern Belle
- Interdimensional Escalations
- Devilish Deals
- Bones of the Mountain (I had played this in an earlier UB version w/o issues)
- West Marches of Solasta - I was able to start the game just fine, but got the instability/no save shortly into it.

Some campaigns where it worked fine:
- Crown for a Krynn
- Tomb of Doom
- Island of Domination
- Lifting the Siege
- Revenge of Xarr-Zuul
- Shire Child
- Morrow's Deep (6)
- Dragons Domain
- Red Crow (6)
- Plight of Ludwig

[Player - 4 - Shire - Premade - UB inst but Off.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15445070/Player.-.4.-.Shire.-.Premade.-.UB.inst.but.Off.log)
[Player -4 - Shire - UB Uninst.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15445071/Player.-4.-.Shire.-.UB.Uninst.log)
[Player - 4 - shire - all premade.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15445072/Player.-.4.-.shire.-.all.premade.log)
[Player - 4 - shire - custom char.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15445073/Player.-.4.-.shire.-.custom.char.log)
[Player - shire - 6 custom char.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15445074/Player.-.shire.-.6.custom.char.log)
[Player - 6 - Shire - After Reinst.log](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/15445075/Player.-.6.-.Shire.-.After.Reinst.log)

**Comment by ThyWoof** [comment]
You are getting errors similar to folks that at some point used an incompatible UMM version. You have to fully delete the UnityModManager folder within the game folder and Re-install the 0.27.8 you have. If you check the log there is a bunch of exceptions trying to load your characters.

**Comment by ThyWoof** [comment]
You also have a bunch of heroes in your host with content that got removed from UB. Need to clean up your entire host as well. These are the tips I can share but nothing wrong with UB.

**Comment by resumeman** [comment]
So I uninstalled UMM and UB and reinstalled them (same versions). I deleted
all characters > lv 1, figuring that any UB artifacts would be on those. I
still got errors.

In the player.log file, all I saw that was obviously obsolete UB was
"cannot find marshall martial..." So I deleted the only character I've ever
played a marshall on. Still get the same notice.

Are there any other specific characters I should delete? Anything else I
should be able to find and remove?
